lib/core/Canvas.js

```diff
@@ -229,8 +228,14 @@ Canvas.prototype._init = function(config) {
     height: '100%'
   });
 
+  assignStyle(svg, {
```

Comment:
[q]: Why do we do this here, rather than via CSS?

Reply:
[Camunda Modeler style](https://github.com/camunda/camunda-modeler/blob/5574871edd34b5ba83ff97bc9d30206456db64e3/client/src/styles/_base.less#L17) was overruling any attempt to set it in diagram-js other than inline like that.

Reply:
I still don't understand. We should offer stable defaults, and allow tools, i.e. Camunda Modeler to do exactly that customization.

If Camunda Modeler chooses to overrule things, then may it be, if the modeler chooses to exclude the canvas from the outline, then a simple `:not(.canvas):focus { ... }` selector is all it needs.